utils/utils.go

```diff
@@ -144,14 +144,26 @@ func StartReaper() {
 		for {
 			// Wait for a child to terminate
 			sig := <-sigs
-			for {
+			for sig == syscall.SIGCHLD {
 				// Reap processes
-				cpid, _ := syscall.Wait4(-1, nil, syscall.WNOHANG, nil)
+				var status syscall.WaitStatus
+				cpid, err := syscall.Wait4(-1, &status, syscall.WNOHANG, nil)
+				if err != nil {
+					if err != syscall.ECHILD {
+						logrus.Infof("wait4: %v", err)
```

Comment:
Hmm, we expect ECHILD if we're calling wait4, but any other error result is unexpected.  Still, I guess we were just ignoring them all before, so :neutral_face:.  Changing it.